Update: Add temporary maintenance route

kevinhinterlong opened the pull request:
![image](https://user-images.githubusercontent.com/12093492/81620731-2efcc280-93b2-11ea-8e68-4827f3f341f5.png)


<!-- The following line must be included in your pull request -->
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

Commit:
Update: Add temporary maintenance route

balajimanoharan commented on app/templates/maintenance.hbs:
```diff
@@ -0,0 +1,61 @@
+{{!-- Copyright 2020, Verizon Media. Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms. --}}
+{{!-- Copyright 2020, Verizon Media. Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms. --}}
```
```suggestion
```

balajimanoharan commented on app/routes/application.js:
```diff
@@ -24,7 +25,10 @@ export default class ApplicationRoute extends Route {
     super.beforeModel(...arguments);
-    return this.intl.setLocale(['en-us']);
+    this.intl.setLocale(['en-us']);
+    if (ENV.environment !== 'test') {
```
Can we feature flag so we can keep this route and reuse in the future?

Commit:
Address Comments, more redirects

- Before you could get around the redirect by loading the
   page, then going directly to Illinois
- Added maintenance mode flag

jkusa commented on tests/acceptance/maintenance-test.js:
```diff
@@ -0,0 +1,106 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, settled } from '@ember/test-helpers';
```
```suggestion
import { visit, currentURL } from '@ember/test-helpers';
```

kevinhinterlong committed:
Update tests/acceptance/maintenance-test.js

Co-authored-by: Jon Kilroy <jkilroy@gmail.com>
